src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -85,6 +85,16 @@ public static class TextLoaderSaverCatalog
             IMultiStreamSource dataSample = null)
             => new TextLoader(CatalogUtils.GetEnvironment(catalog), options, dataSample);
 
+        // 1.4.0 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
```

Comment:
>1.4.0 BACKCOMPAT OVERLOAD -- DO NOT TOUCH [](start = 11, length = 41)

It seems to me that these overloads are not needed, for two reasons:
First of all, since none of the arguments here are optional, it doesn't actually provide backwards compatibility (if a user has code that only uses some of the arguments, it will not compile with this overload). Second of all, since the arguments in the current signature **are** optional, adding another optional argument at the end of the argument list, wouldn't break existing code, so it is backwards compatible. 
Is there a different reason to have these overloads that I am not aware of?